agent/config/runtime.go

```diff
@@ -669,6 +669,12 @@ type RuntimeConfig struct {
 	// and key).
 	EnableAgentTLSForChecks bool
 
+	// EnableCentralServiceConfig controls whether the agent should incorporate
+	// centralized config such as service-defaults into local service registrations.
+	//
+	// hcl: (enable)
```

Comment:
I don't know if there is any value in these hcl comments at all but seems like that should follow format of others.